tests/epyccel/modules/generic_functions_2.py

```diff
@@ -129,14 +129,14 @@ def mix_types_2(x, y):
     return -x
 
 
-@template('g', types=['int[:]', 'real[:]', 'complex[:]'])
+@template('g', types=['int64[:]', 'real[:]', 'complex[:]'])

```

Comment:
Can you explain why this change is necessary? Do we need an issue to correctly determine the precision of `int`?

Reply:
Im trying to see why it fail.

Reply:
Ok. Good luck. The precision of an int is supposed to be different for windows I think. It's determined with `numpy.dtype(int).alignment`. I think we should have `precision = 4` for windows and `precision = 8` for linux/mac. But as long as the array is initialised with the right type then there shouldn't be a problem. Maybe `np.int != int` on windows?